### DISCUSSION TITLE
Privacy & User Data

### DISCUSSION THREAD

**Discussion by jnbht** [OnionUI/Onion#1573] [Q&A] [answered]
Hi, i am very new to OnionOS.
I like it a lot so far. While being in the minority with this, for me privacy and control over my user data ist important.
I did not find any information about this.

Does OnionOS collect or transfer any data back to any servers?
Do you use optimisation tools similar to sth like Google Firebase to get statistics back?
How private is onionOS?

Thanks in advance!!

**Comment by thomasklemenc** [comment, marked as answer]
Hi there, thanks for your questions. As a member of team Onion, and a privacy advocate, I do see your questions as valid and would like to answer them.

To start, we do take your privacy seriously. Onion does not send any data from your device anywhere, unless you actively choose to do so.

> **Does OnionOS collect or transfer any data back to any servers?**

_We collect playtime of games on the device to display them in Activity tracker. This data does never leave the device._
_If you use the built-in rom image scraper, data is transferred to screenscraper.fr API endpoints. This data consists of username, password, game system, rom name, rom size, rom CRC checksum. We do not know how this data is stored/processed on their side of things._

_If you enable logging, data such as when a game is launched or ended and other diagnostic data is stored on your device. None of it leaves the device unless you yourself decide to manually take it off the SD card and send it anywhere._

> **Do you use optimisation tools similar to sth like Google Firebase to get statistics back?**

_No_

> **How private is onionOS?**

_No data ever leaves your device, unless you explicitly want it to. We do not collect any data on users, store or send it anywhere, besides on the device to provide core functionality. We have no infrastructure outside of GitHub that even could collect such data._

I hope this answers your question. Feel free to ask for details, if not. If I noticed I've missed something, I will add it to this comment.

Addendum:

If you use network time synchronization, these endpoints can be contacted to retrieve your time/time zone:

* http://worldtimeapi.org/api/ip.txt
* https://timeapi.io/api/Time/current/zone?timeZone=UTC
* https://api.ipify.org
* https://timeapi.io/api/TimeZone/ip?ipAddress=$ip_address
* time.google.com

Normally, only the first one is contacted. if that fails, it falls over to the next one and so on. Also, this does not happen by default, only if you enable network time sync.

**Comment by jnbht** [comment]
You couldn’t have answered any better. Thank you very much for the detailed information AND for making privacy friendly software. That is not the usual nowadays! ❤️
